src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const clearStyles = styles
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
+  const result = clearStyles.reduce((prev, style) => {
+    const property = style.split(':')[0].trim();
+    const value = style.split(':')[1].trim();
+
+    return { ...prev, [property]: value };
+  }, {});
+
+  return result;
```

Comment:
The function should directly return the result of the reduce operation without assigning it to a variable first, as the variable 'result' is not used elsewhere.